### PR TITLE
Signal through the API that modifications are being made

### PR DESCRIPTION
**Pull request by rinkp**
# Description

1. Improves the notifications (such as pending updates and sync pause) on the home page
2. Introduces notification counter to draw attention to the home page
3. Pauses sync API when decisions are created. 15 minutes is deemed reasonable. If no decisions are entered for more than 15 minutes, the secretary most likely is done entering decisions for that specific meeting.
4. Pauses sync API when decisions are deleted. 60 minutes on deletion to prevent accidentally deleting organs in other systems. Impact of a 60 minute sync delay is considered minimal while hardly any decisions are ever deleted
5. Bugfixes

image

## Related issues/external references
Fixes GH-389.

## Types of changes

- [X] Bug fix _(non-breaking change which fixes an issue)_
- [X] New feature _(non-breaking change which adds functionality)_
- [ ] Breaking change _(fix or feature that would cause existing functionality to change)_
- [ ] Documentation improvement

### CODE REVIEW

**Comment by rinkp** [comment]
Psalm locally:

image

The PHPUnit tests are caused by the Notification helper. It loads `FrontPageService` which needs the `ApiService` which needs the `MemberMapper` from reportdb

**Comment by tomudding** [comment]
PHPStan locally:
image

File: module/Database/view/database/index/index.phtml [hunk 1]
```diff
@@ -11,9 +11,13 @@ use Laminas\View\Renderer\PhpRenderer;
...
  * @var int $prospectives
  * @var int $updates
+ * @var bool $syncPaused
+ * @var DateTime $syncPausedUntil
+ * @var int $totalCount
  */
 
-$showPendingUpdates = 0 < $updates;
+$twoColumnMode = $totalCount > 0;
+$syncPausedDifference = $syncPausedUntil->getTimestamp() - (new DateTime())->getTimestamp();
```
Comment (tomudding) [inline module/Database/view/database/index/index.phtml:20]: `syncPausedUntil` can be `null` (if the specific config item is not yet configured) causing this to throw an error.

File: module/Application/src/Module.php [hunk 2]
```diff
@@ -50,7 +52,8 @@ public function onBootstrap(MvcEvent $e): void
 
         $locale = $this->determineLocale($e);
 
-        $mvcTranslator = $e->getApplication()->getServiceManager()->get(MvcTranslator::class);
+        $serviceManager = $e->getApplication()->getServiceManager();
+        $mvcTranslator = $serviceManager->get(MvcTranslator::class);
```
Comment (tomudding) [inline module/Application/src/Module.php:56]: What is the reason for splitting this?

**Review by tomudding — CHANGES_REQUESTED** [review]
Looks good! I found one minor issue and have one question about something but otherwise this is ready to be merged.

File: module/Application/src/Module.php (hunk 2 above)
Comment (rinkp) [reply module/Application/src/Module.php]: These were remains from an earlier version of the code which reused `$serviceManager`

**Comment by rinkp** [comment]
The phpstan issue seems to be related due to incompatibility between main and the new branch because of renaming database_doctrine_em.

Psalm will no longer complain if `psalm.xml` is modified in a PR

**Review by tomudding — APPROVED** [review]
Awesome! Thank you for your work!

**Pull request merged**